service/dap/server.go

```diff
@@ -464,6 +469,9 @@ func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 		s.args.stackTraceDepth = int(depth)
 	}
 
+	globals, ok := request.Arguments["showGlobalVariables"]
+	s.args.showGlobalVariables = ok && globals == true
```

Comment:
you can replace this with just `s.args.showGlobalVariables, _ = request.Arguments["showGlobalVariables"].(bool)`

Reply:
Agree, that's better, more succinct. However, now that I am thinking about this, I should rewrite this to be conditional. I recently made a change to have the defaults in a struct that I set in the constructor, so I should no longer rely on these expressions producing the right defaults.